src/common-utils/main.sh

```diff
@@ -567,6 +568,55 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
     fi
 fi
 
+# *********************************
+# ** Enable shell history **
+# *********************************
+#
+# Generally, the installation favours configuring a shell to use the mounted volume
+# for storing history rather than symlinking as any operation to recreate files 
+# could delete the symlink. 
+# On shells where this isn't possible, we symlink as a fallback.
+#
+
+if [ "${ALLOW_SHELL_HISTORY}" = "true" ]; then
+# Set HISTFILE for bash
+cat << EOF >> "${user_home}/.bashrc"
+if [[ -z "\$HISTFILE_OLD" ]]; then
+    export HISTFILE_OLD=\$HISTFILE
+fi
+export HISTFILE=/dc/shellhistory/.bash_history
```

Comment:
Would there be value in keeping the default path `~/.bash_history` and instead make that a symbolic link to the actual file in the volume? (I haven't tried using a symbolic link, best to check first if bash/zsh work correctly with that.)

Reply:
done like this. thanks.